### PR TITLE
Add Docker

### PR DESCRIPTION
**Pull request by MoinMarcell**

### CODE REVIEW

**Review by Flooooooooooorian — COMMENTED** [review]
Please dont push your frontend build

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+ENV ENVIROMENT=prod
+
+MAINTAINER Axel Haar <Axel.Haar@hygene.de>
+
+EXPOSE 8080
+
+ADD ./backend/target/backend-0.0.1-SNAPSHOT.jar app.jar
+
+CMD ["sh", "-c", "java -jar /app.jar"]
```
Comment (devBK84) [inline Dockerfile:11]: Geile scheiße...

**Review by devBK84 — COMMENTED** [review]
Good man and great job! :)

**Comment by Flooooooooooorian** [comment]
You still have your frontend build commited... And solve conflicts 👍

File: Dockerfile (hunk 1 above)
Comment (devBK84) [reply Dockerfile:11]: > You still have your frontend build commited... And solve conflicts 👍

Alright mate

**Comment by MoinMarcell** [comment]
> You still have your frontend build commited... And solve conflicts 👍

I cant replay on your comments and i cant resolve the conflict it is "ausgegraut" ^^

**Review by Flooooooooooorian — APPROVED** [review]
As already disscussed there a still changes not relevant to your docker feature but the rest looks great

**Pull request merged**